docs/keyed_services.md

```diff
@@ -0,0 +1,184 @@
+### Keyed Services
+
+Most features should be scoped to a Profile/BrowserContext and be
+BrowserContextKeyedServices per
```

Comment:
I believe it should be `ProfileKeyedServiceFactory` as in the example below.

Reply:
`ProfileKeyedServiceFactory` inherits from `BrowserContextKeyedServiceFactory`- are you saying creating factory for your new keyed service inheriting from `ProfileKeyedServiceFactory` is preferred?

Reply:
yes `ProfileKeyedServiceFactory` is preferred